### DISCUSSION TITLE
Allow manually specifying namespaces

### DISCUSSION THREAD

**Discussion by jetocotoje** [getseabird/seabird#52] [Ideas]
### Summary

In some cases, a user cannot list all namespaces in a cluster. It would be convenient if the user could manually specify which namespaces he has access to, because otherwise Seabird cannot be used for such clusters.

### Motivation

A cluster with some level of hardening may disallow access to list resource "namespaces". Such cluster cannot then be used in the app.

### Proposal

Add an option to manually specify accessible namespaces in such cases.

**Comment by mikk150** [comment]
In my workflow, I actually do not even want to see all namespaces by default (It is nice option to have)

**Comment by d-enk** [comment]
I remind you that without this option this application cannot be used on a real k8s by anyone except admins with full access, who usually use other tools...
